tools/accuracy/accuracy_checker.py

```diff
@@ -159,7 +174,8 @@ def main():
     if args.verbose:
         print(model)
 
-    model.compile(migraphx.get_target('gpu'))
+    if args.ort_run is False:
```

Comment:
how do you run this with ORT ? does ORT generates a compiled model that you pass to the accuracy checker?

Reply:
The `ort_run` flag enables to run of a model through ORT with random data with optional logging.  
From what i understand then, it is not doing any accuracy checking. `accuracy_checker` script may not be the best place in that case.  It is fine for now, but later if you want to compare optimizations between different EPs and MIGraphX EPs, I would rather do it in a separate script later instead of injecting it here.

Reply:
I agree with some of what you mean here, but the intent here is down this would fit in the existing workflow as we can now invoke different EP as well to test accuracy relative to MIGraphX as well. Fallback performance vs standalone vs a pure MIGraphX run through our API can now be benchmarked if we were to add additional flags on top of these changes

I think the only contentious point is the ability to sanity check whether a model will even run correctly on ORT standalone in this program vs splitting.  I think that would be useful to have this here vs split it  as this tool would become a swiss army knife tool of comparison between both API's workflows that MIGraphX interfaces if a customer doesn't want to use our driver solution. Again, this is a bigger discussion in the future.